evalml/tests/component_tests/test_imputer.py

```diff
@@ -291,3 +291,44 @@ def test_imputer_with_none():
     imputer = Imputer()
     transformed = imputer.fit_transform(X, y)
     assert_frame_equal(transformed, expected, check_dtype=False)
+
+
+@pytest.mark.parametrize("data_type", ['pd', 'ww'])
+def test_imputer_all_bool(data_type):
```

Comment:
Since `Imputer` uses `SimpleImputer` under the hood, and you're putting the fix into `SimpleImputer`, I think the coverage of `SimpleImputer` is enough and you can delete this copy of the unit test.

Reply:
Yep, my understanding is that while `Imputer` does use `SimpleImputer` under the hood, the output of `Imputer` is different as it recasts everything to the proper type. I don't see an issue with changing this in the future.